default/species.txt

```diff
@@ -1542,6 +1560,25 @@ Species
 //####                                               ####\\
 //#######################################################\\
 
+// BROAD_EP and NARROW_EP are stub placeholders for applying the
+// board/narrow environment tolerance description to a species.
+BROAD_EP
+'''
+        EffectsGroup
+            description = "BROAD_EP"
+            scope = Source
+            activation = None
+            effects = SetPlanetType type = Target.OriginalType
```

Comment:
This EffectsGroup should not exist in the first place.  It is only a workaround for the missing feature of attaching a description to an environmental preference.